index.html

```diff
@@ -76,6 +76,33 @@ <h2>Help your friends find some good learning resources</h2>
         </div>
 
         <div class="grid">
+            <!-- ________ Abhay Chandel Contributor card START ________  -->
+            <div class="card">
+                <p class="name">Abhay Chandel</p>
+                <p class="contact">
+                    <i class="fab fa-twitter"></i>
+                    <a href="https://twitter.com/abhaychandel404" target="_blank">@abhaychandel404</a>
+                </p>
+                <p class="about">I am currently pursuing a Bachelor of Technology in Computer Science Engineering. I am currently
+                    studying JavaScript and DSA. I am gonna learn React after completing Js.</p>
+                <div class="resources">
+                    <p>3 Useful Dev Resources</p>
+                    <ul>
+                        <li>
+                            <a href="https://www.freecodecamp.org" target="_blank"
+                                title="First Resource">FreeCodeCamp</a>
+                        </li>
+                        <li>
+                            <a href="https://www.edx.org" target="_blank"
+                                title="Second Resource">Edx Courses</a>
+                        </li>
+                        <li>
+                            <a href="https://www.edx.org" target="_blank" title="Third resource">The Odin Project</a>
```

Comment:
Change the link of The Odin Project from "edx" link to "odin project" link

Reply:
I have changed it please take a view again sorry for that mistake